src/EditorFeatures/CSharpTest/RefactoringHelpers/RefactoringHelpersTests.cs

```diff
@@ -527,6 +527,22 @@ public class Class1
             await TestMissingAsync<MethodDeclarationSyntax>(testText);
         }
 
+        [Fact]
+        [WorkItem(37837, "https://github.com/dotnet/roslyn/issues/37837")]
+        public async Task TestEmptyParameter()
```

Comment:
there seems no test for attribute case you added?

Reply:
Doesn't really matter, we're only working on syntax tree & to trigger the issue the method's decl has to be incomplete anyway (so the code can't compile anyway).